backend/src/main/resources/api.yaml

```diff
@@ -131,21 +131,231 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/FilledPoll'
+              $ref: "#/components/schemas/FilledPoll"
         required: true
       responses:
-        202:
+        201:
           description: Answers submitted succesfully.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/FilledPoll"
+        400:
+          description: Invalid request.
+        409:
```

Comment:
Jak _no longer existst_ to raczej bym `404` zwrócił po prostu

Reply:
W sumie nam tej 404 brakuje jak takiego polla nigdy nie było tak samo, można to pod jedno podłączyć